{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/cli.py

```diff
@@ -0,0 +1,84 @@
+from pathlib import Path
+
+from jinja2 import Template
+import click
+import re
+
+from vivarium_gbd_access.gbd import ARTIFACT_FOLDER
+from vivarium_cluster_tools.psimulate.utilities import get_drmaa
+
+JOB_MEMORY_NEEDED = 50
+JOB_TIME_NEEDED = '24:00:00'
+
+drmaa = get_drmaa()  # safe if not on cluster
+
+
+def create_and_run_job(model_spec_path: Path, output_root: Path):
+    with drmaa.Session() as s:
+        jt = s.createJobTemplate()
+        jt.remoteCommand = "build_artifact"
+        jt.nativeSpecification = '-V -l m_mem_free={}G,fthread=1,h_rt={} -q all.q -P proj_cost_effect'.format(
+            JOB_MEMORY_NEEDED, JOB_TIME_NEEDED)
+        jt.args = [model_spec_path, '-o', output_root]
+        jt.jobName = f'build_artifact_{model_spec_path.name}'
+        result = s.runJob(jt)
+        print(f"Submitted job for {model_spec_path.name}. Job id: {result}")
+
+
+@click.command()
+@click.option('--model_spec', '-m', multiple=True, type=click.Path(dir_okay=False, exists=True),
+              help='Multiple model spec files can be provided. Each requires the option switch.')
+@click.option('--project_name', default='vivarium_conic_sam_comparison',
```

Comment:
You shouldn't need to get the project here.  It should come vadmin init and be templated in this repo {{cookiecutter.package_name}}